foundation-extensions/mobile-core/configuration/README.md

```diff
@@ -397,7 +397,7 @@ Some extension developers might use different configuration values based on thei
 ```
 
 {% hint style="success" %}
-Each time a remote configuration is generated by Experience Platform Launch, a `build.environment` value is set. This value is based on the environment that you are publishing. When the remote configuration is downloaded, the Configuration extension considers the value in `build.environment` and provides **only** the non-prefixed version for the current environment in the shared state.
+Each time a remote configuration is generated by Platform, a `build.environment` value is set. This value is based on the environment that you are publishing. When the remote configuration is downloaded, the Configuration extension considers the value in `build.environment` and provides **only** the non-prefixed version for the current environment in the shared state.
```

Comment:
Each time a remote configuration is generated in the Data Collection UI, a `build.environment` value is set.